playground/AzureServiceBus/ServiceBusWorker/Program.cs

```diff
@@ -0,0 +1,19 @@
+using ServiceBusWorker;
+
+var builder = Host.CreateApplicationBuilder(args);
+
+builder.AddServiceDefaults();
+
+builder.AddAzureServiceBusClient("sbemulator");
+
+Console.WriteLine("ServiceBus producer/consumer test");
+
+builder.Services.AddHostedService<Consumer>();
+Console.WriteLine("Starting Service Bus consumer...");
```

Comment:
(nit) these console writelines are kind of incorrect. All we did is add a service, but it is saying "starting...".

Can we just remove the writelines?